sources/packages/web/src/views/aest/SearchStudents.vue

```diff
@@ -2,32 +2,48 @@
   <full-page-container>
     <h2 class="color-blue">Search Students</h2>
     <v-row class="mt-5"
-      ><v-col><label for="appNumber">Application Number</label></v-col
-      ><v-col><label for="firstName">Given Names</label></v-col
-      ><v-col><label for="lastName">Last Name</label></v-col> <v-col></v-col
+      ><v-col>Application Number</v-col><v-col>SIN</v-col
```

Comment:
Can you please review the layout and use components like `body-header` and `content-group` to make it look like more close to Figma.
![image](https://user-images.githubusercontent.com/61259237/178556500-b1d9595e-b8f8-489c-a6b5-25d90723a08b.png)